katamari/com.hack_computer.Clippy.Extension.json.in

```diff
@@ -3,7 +3,7 @@
     "branch": "${BRANCH}",
     "runtime": "com.hack_computer.Clubhouse",
     "runtime-version": "${BRANCH}",
-    "sdk": "org.gnome.Sdk//3.36",
+    "sdk": "org.gnome.Sdk//41",
```

Comment:
Huh, I've never seen this syntax used in this field before. I guess I have not spent much time with `build-extension: true`.